src/blueprint/blueprint.rs

```diff
@@ -75,65 +81,66 @@ impl Definition {
   }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct InterfaceTypeDefinition {
   pub name: String,
   pub fields: Vec<FieldDefinition>,
   pub description: Option<String>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct ObjectTypeDefinition {
   pub name: String,
   pub fields: Vec<FieldDefinition>,
   pub description: Option<String>,
   pub implements: BTreeSet<String>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct InputObjectTypeDefinition {
   pub name: String,
   pub fields: Vec<InputFieldDefinition>,
   pub description: Option<String>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct EnumTypeDefinition {
   pub name: String,
   pub directives: Vec<Directive>,
   pub description: Option<String>,
   pub enum_values: Vec<EnumValueDefinition>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct EnumValueDefinition {
   pub description: Option<String>,
   pub name: String,
   pub directives: Vec<Directive>,
 }
 
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Debug, Default, Serialize)]
 pub struct SchemaDefinition {
   pub query: String,
   pub mutation: Option<String>,
   pub directives: Vec<Directive>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct InputFieldDefinition {
   pub name: String,
   pub of_type: Type,
   pub default_value: Option<serde_json::Value>,
   pub description: Option<String>,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Serialize)]
 pub struct Cache {
   pub max_age: NonZeroU64,
+  #[serde(skip)]
```

Comment:
This is ok to skip. We are moving this logic out of the blueprint anyways.

Reply:
@tusharmath you mean the entire Cache or just what I've skipped?

Reply:
Just what you have. We have another PR where this will be removed.